toolkit/tools/imageconfigvalidator/imageconfigvalidator_test.go

```diff
@@ -90,5 +100,6 @@ func TestShouldFailDeeplyNestedParsingError(t *testing.T) {
 			return
 		}
 	}
-	assert.Fail(t, "Could not find 'core-efi.json' to test")
+	assert.Fail(t, "Could not find "+targetPackage+" to test")
+}
```

Comment:
Does this compile? I cannot find a new opening bracket.

Reply:
Fixed, added it back in PR 4/5

---
In reply to: [549908287](https://github.com/microsoft/CBL-Mariner/pull/480#discussion_r549908287) [](ancestors = 549908287)